### PR TITLE
[zh] Resync configure-liveness-readiness-startup-probes

### PR DESCRIPTION
**Pull request by tengqm**
Related: #26178

### CODE REVIEW

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 1]
```diff
@@ -512,23 +512,71 @@ Defaults to 3. Minimum value is 1.
   就绪探测情况下的放弃 Pod 会被打上未就绪的标签。默认值是 3。最小值是 1。
 
 <!--
+Before Kubernetes 1.20, the field `timeoutSeconds` was not respected for exec probes:
+probes continued running indefinitely, even past their configured deadline,
+until a result was returned.
+-->
+在 Kubernetes 1.20 版本之前，exec 探针会忽略 `timeoutSeconds`：探针会无限期地
```
Comment (ydFu) [inline content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:519]:
```suggestion
在 Kubernetes 1.20 版本之前，exec 探针会忽略 `timeoutSeconds`：
```

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 2]
```diff
@@ -512,23 +512,71 @@ Defaults to 3. Minimum value is 1.
...
+继续运行，甚至可能超过所配置的限期，直到返回结果为止。
```
Comment (ydFu) [inline content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:520]:
```suggestion
探针会无限期地继续运行，甚至可能超过所配置的限期，
直到返回结果为止。
```

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 3]
```diff
@@ -512,23 +512,71 @@ Defaults to 3. Minimum value is 1.
...
+（将其设置为 `false`），从而恢复之前版本中的运行行为，之后当集群中所有的
+exec 探针都设置了 `timeoutSeconds` 参数后，移除此标志重载。
+如果你有 Pods 受到此默认 1 秒钟超时值的影响，你应该更新 Pod 对应的探针的
+超时值，这样才能为最终去除该特性门控做好准备。
+
+<!--
+With the fix of the defect, for exec probes, on Kubernetes `1.20+` with the `dockershim` container runtime,
+the process inside the container may keep running even after probe returned failure because of the timeout.
+-->
+当此缺陷被修复之后，在使用 `dockershim` 容器运行时的 Kubernetes `1.20+`
```
Comment (ydFu) [inline content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:545]:
```suggestion
当此缺陷被修复之后，在使用 `dockershim` 容器运行时的 Kubernetes `1.20+`版本中，对于 exec 探针而言，
```

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 4]
```diff
@@ -512,23 +512,71 @@ Defaults to 3. Minimum value is 1.
...
+版本中，对于 exec 探针而言，容器中的进程可能会因为超时值的设置保持持续运行，
```
Comment (ydFu) [inline content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:546]:
```suggestion
容器中的进程可能会因为超时值的设置保持持续运行，即使探针返回了失败状态。
```

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 5]
```diff
@@ -512,23 +512,71 @@ Defaults to 3. Minimum value is 1.
...
+即使探针返回了失败状态。
```
Comment (ydFu) [inline content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:547]:
```suggestion
```

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 6]
```diff
@@ -512,23 +512,71 @@ Defaults to 3. Minimum value is 1.
...
+This defect was corrected in Kubernetes v1.20. You may have been relying on the previous behavior,
+even without realizing it, as the default timeout is 1 second.
+As a cluster administrator, you can disable the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) `ExecProbeTimeout` (set it to `false`)
+on each kubelet to restore the  behavior from older versions, then remove that override
+once all the exec probes in the cluster have a `timeoutSeconds` value set.  
+If you have pods that are impacted from the default 1 second timeout,
+you should update their probe timeout so that you're ready for the
+eventual removal of that feature gate.
+-->
+这一缺陷在 Kubernetes v1.20 版本中得到修复。你的探针行为可能一直如此，甚至
```
Comment (SataQiu) [inline content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:532]:
```suggestion
这一缺陷在 Kubernetes v1.20 版本中得到修复。你可能一直依赖于之前错误的探测行为，甚至
```

**Review by SataQiu — COMMENTED** [review]
/approve

**Pull request merged**